test/integration/integration.cc

```diff
@@ -286,57 +294,49 @@ void BaseIntegrationTest::registerTestServerPorts(const std::vector<std::string>
   FakeHttpConnectionPtr fake_upstream_connection;
   IntegrationStreamDecoderPtr response(new IntegrationStreamDecoder(*dispatcher_));
   FakeStreamPtr request;
-  executeActions({[&]() -> void { codec_client = makeHttpConnection(std::move(conn), type); },
-                  [&]() -> void {
-                    Http::TestHeaderMapImpl headers{{":method", "POST"},
-                                                    {":path", "/test/long/url"},
-                                                    {":scheme", "http"},
-                                                    {":authority", "host"},
-                                                    {"x-lyft-user-id", "123"},
-                                                    {"x-forwarded-for", "10.0.0.1"}};
-                    if (big_header) {
-                      headers.addViaCopy("big", std::string(4096, 'a'));
-                    }
-
-                    codec_client->makeRequestWithBody(headers, request_size, *response);
-                  },
-                  [&]() -> void {
-                    fake_upstream_connection =
-                        fake_upstreams_[0]->waitForHttpConnection(*dispatcher_);
-                  },
-                  [&]() -> void { request = fake_upstream_connection->waitForNewStream(); },
-                  [&]() -> void { request->waitForEndStream(*dispatcher_); },
-                  [&]() -> void {
-                    request->encodeHeaders(Http::TestHeaderMapImpl{{":status", "200"}}, false);
-                    request->encodeData(response_size, true);
-                  },
-                  [&]() -> void { response->waitForEndStream(); },
-                  // Cleanup both downstream and upstream
-                  [&]() -> void { codec_client->close(); },
-                  [&]() -> void { fake_upstream_connection->close(); },
-                  [&]() -> void { fake_upstream_connection->waitForDisconnect(); }});
+  executeActions(
+      {[&]() -> void { codec_client = makeHttpConnection(std::move(conn), type, version); },
+       [&]() -> void {
+         Http::TestHeaderMapImpl headers{{":method", "POST"},
+                                         {":path", "/test/long/url"},
+                                         {":scheme", "http"},
+                                         {":authority", "host"},
+                                         {"x-lyft-user-id", "123"},
+                                         {"x-forwarded-for", "10.0.0.1"}};
+         if (big_header) {
+           headers.addViaCopy("big", std::string(4096, 'a'));
+         }
+
+         codec_client->makeRequestWithBody(headers, request_size, *response);
+       },
+       [&]() -> void {
+         fake_upstream_connection = fake_upstreams_[0]->waitForHttpConnection(*dispatcher_);
+       },
+       [&]() -> void { request = fake_upstream_connection->waitForNewStream(); },
+       [&]() -> void { request->waitForEndStream(*dispatcher_); },
```

Comment:
How come so much churn in formatting? It's a bit harder to pick out the diff here.